packages/amplify_core/lib/src/types/query/query_field_operators.dart

```diff
@@ -79,6 +79,22 @@ abstract class QueryFieldOperator<T> {
     // TODO sanitize other types appropriately
     return value;
   }
+
+  /// Gets the correct field to use as a comparison for nested models.
+  ///
+  /// TODO: Remove when codegen supports a `toMap()` method.
```

Comment:
This would be replaced or updated if the codegen changes in the issue description were added.